scripts/ci/test_ws12_workflow_contracts.py

```
@@ -1147,6 +1149,56 @@ def setUp(self) -> None:
+                    errors,
+                )
+
+    def test_hooks_parity_timeout_ignores_commented_and_unrelated_decoys(self) -> None:
+        workflow = self.workflows[PLATFORM_WORKFLOW]
+        backslash = "\\"
+        live_timeout = f"timeout --signal=INT --kill-after=30s 25m {backslash}"
+        decoys = (
+            (
+                "commented",
+                "# timeout --signal=INT --kill-after=30s 25m cargo test\n"
+                f"          cargo test {backslash}",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)